src/base_cache.cpp

```diff
@@ -0,0 +1,8 @@
+#include "base_cache.hpp"
+
+BaseCache::BaseCache(const CachePropertyStruct &setting) {
+    property = setting;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'property' is assigned in constructor body. Consider performing initialization in initialization list.](https://app.codacy.com/manual/jerry0715no14/Cache_sim/pullRequest?prid=4735708)

Reply:
oops